charts/fleet/Chart.yaml

```diff
@@ -8,7 +8,7 @@ version: v6.6.9
 home: https://github.com/fleetdm/fleet
```

Comment:
the script used to bump the version... wonder why it hasn't been catching it lately... I can peek later